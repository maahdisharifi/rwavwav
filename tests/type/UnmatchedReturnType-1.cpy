double returnDoubleAsInt():
    return 42
end

int returnIntAsDouble():
    return 3.14
end

int main():
    double d = returnDoubleAsInt()
    int i = returnIntAsDouble()
    return 0
end