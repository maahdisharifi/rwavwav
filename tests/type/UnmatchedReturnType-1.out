Line:6 -> return type mismatch